Sources/SwiftProtobuf/BinaryDelimited.swift

```diff
@@ -99,9 +99,9 @@ public enum BinaryDelimited {
   ///     extensions in this message or messages nested within this message's
   ///     fields.
   ///   - partial: If `false` (the default), this method will check
-  ///     `Message.isInitialized` before encoding to verify that all required
+  ///     `Message.isInitialized` after decoding to verify that all required
```

Comment:
You changed this one to "after decoding", but the others below just changed "decoding" so they're "before decoding". Should those also be "after"?

Reply:
Ugh, just created #1445 to hopefully get more things correct on `main`, once that lands I'll cherry-pick that into this.

Reply:
Second cherry-pick added.